docs/sphinx/_toc.yml.in

```diff
@@ -40,8 +40,14 @@ subtrees:
       title: API libraries
     - file: reference/rocm-tools.md
       title: Tools
-    - file: reference/gpu-arch-specs.rst
-      title: Hardware specifications
+    - file: reference/env-variables.rst
+      title: Environment variables
+    - file: reference/gpu-arch.rst
+      title: GPU architectures
+      subtrees:
+      - entries:
+        - file: reference/gpu-arch/gpu-arch-spec-overview.rst
+          title: Hardware specifications overview
```

Comment:
Same as for the index: Please don't break these links, and leave them as is. They don't even show up in the side bar right now, because these files don't exist.

Reply:
The extension is wrong. It should be like reference/env-variables.